setup.py

```diff
@@ -174,10 +174,14 @@ def vitables_data_files():
 # main stuff #
 # ========== #
 
+execfile('./liam2/version.py')
+# now we have a `__version__` variable
+
+
 setup(
     name="liam2",
     # cx_freeze wants only ints and dots (full version number)
-    version=int_version('0.11.0'),
+    version=int_version(__version__[:-2] if __version__.endswith('.0') else __version__),

```

Comment:
you misunderstood my comment. The full version is needed here, but I prefer the "short" version to be displayed at https://github.com/liam2/liam2/blob/master/liam2/main.py#L345